internal/ibmcloud-janitor/resources/vpc_networks.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+type VpcNetwork struct{}
+
+// Cleans up the FIPs, subnets and its attached gateways in a given region
+func (VpcNetwork) cleanup(options *CleanupOptions) error {
```

Comment:
and also it will be helpful to add more comments with mentioning step1, step2, step3 etc.. e.g: step1: delete publicgateway, step2: floating point ip etc..

Reply:
you also might need to handle the load balancers

Reply:
I'm planning to handle the load balancer cleanup in a follow up PR(since we are using the standard cluster template in the CAPI e2e tests as of now). Does it sound okay?